sdk/feature-management/src/featureManager.ts

```diff
@@ -179,12 +182,73 @@ export class FeatureManager implements IFeatureManager {
             }
         }
 
+        // The callback will only be executed if telemetry is enabled for the feature flag
+        if (featureFlag.telemetry?.enabled && this.#onFeatureEvaluated !== undefined) {
+            this.#onFeatureEvaluated(result);
+        }
+
         return result;
     }
 }
 
-interface FeatureManagerOptions {
+export interface FeatureManagerOptions {
```

Comment:
what's the purpose of code move? to put exported types together? Is there any change other than that?

Reply:
public stuff should be placed in front of private stuff. I follow this practice in .NET